### PR TITLE
Fix bug 1325125: expand blocked protocols regex to include preceding spaces

### PR DESCRIPTION
**Pull request by claudijd**
Expand regex to match on preceding spaces

### CODE REVIEW

File: kuma/settings/common.py [hunk 1]
```diff
@@ -1185,7 +1185,7 @@ def _get_locales():
         "are removed from the file storage"
     ),
     KUMA_WIKI_HREF_BLOCKED_PROTOCOLS=(
-        '(?i)^(data\:?)',
+        '(?i)^\s*(data\:?)'
```
Comment (jgmize) [inline kuma/settings/common.py]: Missing `,` after regex.

Comment (claudijd) [inline kuma/settings/common.py]: Fixed in 4b6deb3a31f1a993019eddd9fae9b099b8be000f

**Comment by jwhitlock** [comment]
Replaces with PR #4083. @claudijd is still credited, but includes tests and expanded matching in the ``trackOutboundLinks`` JS

**Pull request closed without merging**